src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -138,8 +138,10 @@ export const SystemCVEs = ({
     useEffect(() => {
         if (isFirstLoad) {
             if (setPageTitle) {
-                chrome.updateDocumentTitle(`${entity.display_name}
-                - ${intl.formatMessage(messages.systemsHeader)} - ${intl.formatMessage(messages.pageTitleSuffix)}`);
+                chrome.updateDocumentTitle(
+                    `${entity.display_name} - Systems - Vulnerability | RHEL`,
+                    true
```

Comment:
Instead of adding the suffix manually here and disabling chrome adding a suffix, we can change the suffix for all apps in chrome itself: https://github.com/RedHatInsights/insights-chrome/blob/03a0a14126a88ece38d34ad24b43692f67a5e662/src/hooks/useBundle.ts#L22